multichain-testing/tools/e2e-tools.js

```diff
@@ -23,8 +23,8 @@ import { makeTracer } from '@agoric/internal';
 const trace = makeTracer('E2ET');
 
 // The default of 6 retries was failing.
-// XXX even 15 wasn't enough
-const SMART_WALLET_PROVISION_RETRIES = 30;
+// XXX also tried 15, 30. There's probably something deeper to fix.
+const SMART_WALLET_PROVISION_RETRIES = 100;
```

Comment:
has a higher retry count ever helped?

my intuition says that the thing gets into a state where just retrying won't help.

Reply:
I think so but since it's intermittent it's hard to know for sure. If 100 doesn't work then I'll take a different tack.